desc/compute/_neoclassical.py

```diff
@@ -235,7 +249,7 @@ def fun(pitch_inv):
                 bounce.points(pitch_inv, num_well),
                 is_fourier=True,
             )
-            return safediv(H**2, I).sum(axis=-1).mean(axis=-2)
+            return safediv(H**2, I).sum(-1).mean(-2)
```

Comment:
for future, I like the verbosity of keeping axis there... fine with this but just saying this is not really a needed change